src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -4,12 +4,15 @@
 import java.awt.event.ActionListener;
 
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.ExtensionFeature;
 
 public class SelectDriverMenuOptionListener implements ActionListener {
 	private DriverManager driverManager;
 	private Job2dDriver driver;
+	private ExtensionsManager extensionsManager;
 
 	public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
+		this.extensionsManager = ExtensionFeature.getExtensionsManager();
```

Comment:
To keep the same convention as for other fields, pass the extensionsManager as a param (and remove dependency to ExtensionFeature)